experiments/conductor/cmd/runner/runner.go

```diff
@@ -116,27 +118,27 @@ func BuildRunnerCmd() *cobra.Command {
 		"v", false, "Enable verbose output logging")
 	cmd.Flags().StringVarP(&opts.branchSuffix, "branch-suffix",
 		"", "", "Suffix to append to remote branch names when pushing")
+	cmd.Flags().BoolVarP(&opts.skipMakeReadyPR, "skip-makereadypr",
+		"", false, "Skip the make ready-pr step when pushing branches")
 
 	return cmd
 }
 
 type RunnerOptions struct {
-	branchConfFile string
-	branchRepoDir  string
-	command        int64
-	loggingDir     string
-	timeout        time.Duration
-	readFileType   string
-	defaultRetries int    // Default number of retries for commands
-	forResources   string // Comma-separated list of branch names to filter on
-
-	// forResourcesRegex filters branches, only branches that match the regex are processed
-	forResourcesRegex string
-
-	force        bool   // Force flag to override file existence checks
-	numCommits   int    // Number of commits to diff (default: 1)
-	verbose      bool   // Verbose output flag
-	branchSuffix string // Suffix to append to remote branch names when pushing
+	branchConfFile    string
+	branchRepoDir     string
+	command           int64
+	loggingDir        string
+	timeout           time.Duration
+	readFileType      string
+	defaultRetries    int    // Default number of retries for commands
+	forResources      string // Comma-separated list of branch names to filter on
+	forResourcesRegex string // Regex to filter branches, only branches that match the regex are processed
+	force             bool   // Force flag to override file existence checks
+	numCommits        int    // Number of commits to diff (default: 1)
+	verbose           bool   // Verbose output flag
+	branchSuffix      string // Suffix to append to remote branch names when pushing
+	skipMakeReadyPR   bool   // Skip make ready-pr step when pushing branches
```

Comment:
I feel the default behavior should be to "skip"? In other words, the flag should be named "makeReadyPR," and developers opt in by setting it to true. I personally only perform manual pushes, but it might be worth checking with those who use this command regularly.

Reply:
We can change the defaulting later. I found this to be useful

Reply:
Yeah I can see either way. This is not a blocker.